sinobuz.html

```diff
@@ -101,83 +101,20 @@
                          tooltip : "Hold VEFX and Effect during a song to restart",
                          patches : [{offset : 0x4e284, off: [0x8A, 0xC3], on : [0xB0, 0x01]}]
                         },
-                    ], "stock"),
-
-                    new DllPatcher("bm2dx_omni", [
                         {
-                         name : "Timer Freeze",
-                         patches : [{offset : 0x9BCFE, off: [0x74], on : [0xEB]}]
-                        },
-                        {
-                         name : "Premium Free",
-                         patches : [{offset : 0x60B9B, off: [0x75], on : [0xEB]}]
-                        },
-                        {
-                         name : "Premium Free (2 player mode)",
-                         patches : [{offset : 0x60D14, off: [0x74, 0x2f], on : [0x90, 0x90]},
-                                    {offset : 0x60D25, off: [0x0f, 0x85, 0x3b], on : [0xe9, 0x3c, 0xff]},
-                                    {offset : 0x60D2A, off: [0xff], on : [0x90]}]
-                        },
-                        {
-                         name : "Premium Free Timer Freeze",
-                         patches : [{offset : 0x5DFFD, off: [0x48], on : [0x90]}]
-                        },
-                        {
-                         name : "Unlock All Songs",
-                         patches : [{offset : 0x5C125, off: [0x74, 0x15], on : [0x90, 0x90]}]
+                            //by nibs
+                            name: "Shorter monitor check",
+                            tooltip: "Runs for 300 frames (5 seconds) instead of 1200 (20 seconds), recommended only if you have a very stable framerate",
+                            patches: [{offset: 0x81A5A, off: [0xB0, 0x04], on: [0x2C, 0x01]}]
                         },
                         {
-                         name : "Cursor lock",
-                         tooltip : "After song finishes, song select remains on previous song",
```

Comment:
Can these really be merged?

Reply:
I don't know where these patches came from or what DLL it's for, but the SINOBUZ 7.1 omnimix DLL works fine with just the regular patches